Source/Other/Utility/SPFunctions.m

```diff
@@ -47,6 +47,16 @@ void SPMainQSync(void (^block)(void))
 	}
 }
 
+void delayCallback(SAVoidCompletionBlock callback, double delayInSeconds)
```

Comment:
I know I'm annoying but could we create DispatchQueueExtension.swift file and put this there in swift with optional callback ? :)) More in Swift the better!

Or if it's right now used only on one place, don't use it at all...

Reply:
No joke ... i tried!

```swift
/// Runs code with delay given in seconds. Uses the main thread if not otherwise specified.
///
/// - Parameters:
///   - delayTime: The duration of the delay. E.g. `.seconds(1)` or `.milliseconds(200)`.
///   - qosClass: The global QOS class to be used or `nil` to use the main thread. Defaults to `nil`.
///   - closure: The code to run with a delay.
public func delay(by delayTime: DispatchTimeInterval, qosClass: DispatchQoS.QoSClass? = nil, _ closure: @escaping () -> Void) {
    let dispatchQueue = qosClass != nil ? DispatchQueue.global(qos: qosClass!) : DispatchQueue.main
    dispatchQueue.asyncAfter(deadline: DispatchTime.now() + delayTime, execute: closure)
}
```

Yes, I wanted to use a notification, but this was quicker.

Reply:
> I know I'm annoying but could we create DispatchQueueExtension.swift file and put this there in swift with optional callback ? :)) More in Swift the better!
> 
Something something "'DispatchQueue' is only visible via the Objective-C runtime"